src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/delete_interaction_choice_set_request.cc

```diff
@@ -53,9 +55,14 @@ DeleteInteractionChoiceSetRequest::DeleteInteractionChoiceSetRequest(
                          application_manager,
                          rpc_service,
                          hmi_capabilities,
-                         policy_handler) {}
+                         policy_handler)
+    , response_result_code_(
+          hmi_apis::Common_Result::eType::UNSUPPORTED_RESOURCE)
```

Comment:
@IGapchuk not sure that unsupported resource good initial value. Suggest INVALID_ENUM

Reply:
@AKalinich-Luxoft Changed in e32b004ae4d7e84e153d6b8082cb3c1a800a41ec.